source/security.txt

```diff
@@ -0,0 +1,41 @@
+.. _security:
+
+=======================================================================
+Securely Store an {+atlas-cli+} Configuration File
+=======================================================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The {+atlas-cli+}, like most popular CLIs, enables you to store your configuration file in 
+a secure manner. The following methods are available for securely storing your |service| credentials 
+and configuration file. The configuration file is only accessible to the current user.
+
+Overriding the Private API Key Value by Using an Environment Variable
+---------------------------------------------------------------------
+
+You can specify configuration settings by using :ref:`environment variables <atlas-cli-env-vars>`. 
+When you run any command, each setting that you specify with an environment variable takes precedence 
+over the profile stored in the configuration file. 
+
+You can encrypt the value of your private API key by overriding your private API key value through 
+the :ref:`MONGODB_ATLAS_PRIVATE_API_KEY environment variable <atlas-cli-env-vars>`.
```

Comment:
this guide is about storing the file location but then the first recommendation is not not use the file, I find this a bit confusing